cli/cmd/inject.go

```diff
@@ -40,6 +40,7 @@ var (
 	proxyControlPort    uint
 	proxyMetricsPort    uint
 	proxyAPIPort        uint
+	proxyBindTimeout    string
```

Comment:
This variable should now instead be added to the `proxyConfigOptions` struct in `cli/cmd/root.go`.